torch/testing/_internal/custom_op_db.py

```diff
@@ -177,97 +177,87 @@ def sample_inputs_numpy_view_copy(opinfo, device, dtype, requires_grad, **kwargs
     result = make_arg(2, 3, 4, low=0.9, high=2)
     yield SampleInput(result, args=([2, 12],))
 
-@custom_ops.custom_op('_torch_testing::numpy_cat')
+@torch.library.custom_op('_torch_testing::numpy_cat', mutates_args=())
```

Comment:
As best I can tell, the changes here are just... refactoring? You never use the new decorator here

Reply:
We register an autograd.Function to the Autograd dispatch key. In this PR, we apply the supports_tensorlist to said autograd.Function. This op, using the new custom ops API, would not have passed the new tests without the new decorator.